services/skus/credentials.go

```diff
@@ -688,7 +688,7 @@ func (s *SigningOrderResultErrorHandler) Handle(ctx context.Context, message kaf
 // - create repos for credentials;
 // - move the corresponding methods there;
 // - make those methods work on per-item basis.
-func (s *Service) DeleteOrderCreds(ctx context.Context, orderID uuid.UUID, isSigned bool) error {
+func (s *Service) DeleteOrderCreds(ctx context.Context, orderID uuid.UUID, reqID *uuid.UUID, isSigned bool) error {
```

Comment:
question: is there a specific reason we are passing `reqID` as a pointer should this not be a value?

It suggest that it is going to be modified in some way

Reply:
Plus a lot to this.

Suggestion: make it `uuid.UUID` and make the caller pass either a normal uuid, or `uuid.Nil`.

Reply:
Updated by the patch #2314.